cypress.config.ts

```diff
@@ -7,8 +7,7 @@ export default defineConfig({
     supportFile: "cypress/support/index.ts",
     specPattern: "cypress/integration/**/*.ts",
     experimentalStudio: true,
-    experimentalSessionAndOrigin: true,
-    testIsolation: "legacy",
+    testIsolation: false,
```

Comment:
Is it worth it to file a ticket to do work to be able to turn this on? I do think it would be good to follow their best practices, even if the behavior is not what we are used to.

Reply:
While I do agree tests should be isolated across test suites. I don't necessarily agree with test isolation on a per-test basis for e2e tests. The nature of our apps involve setting up complex state through a series of interactions and it gives us the opportunity to make assertions along the way. Having to start on a clean slate for each test would require us to boilerplate and repeat all the necessary commands to get to the point of an assertion every single time. 

I added this to the UI roundtable agenda to discuss with the team.